iceoryx_hoofs/include/iceoryx_hoofs/internal/log/platform_building_blocks/console_logger.inl

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2021 - 2022 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef IOX_HOOFS_PLATFORM_BUILDING_BLOCKS_CONSOLE_LOGGER_INL
+#define IOX_HOOFS_PLATFORM_BUILDING_BLOCKS_CONSOLE_LOGGER_INL
+
+#include "iceoryx_hoofs/log/platform_building_blocks/console_logger.hpp"
+
+namespace iox
+{
+namespace pbb
+{
+template <uint32_t N>
+// NOLINTJUSTIFICATION see at declaration in header
+// NOLINTNEXTLINE(hicpp-avoid-c-arrays, cppcoreguidelines-avoid-c-arrays)
+inline constexpr uint32_t ConsoleLogger::bufferSize(const char (&)[N]) noexcept
+{
+    return N;
+}
+
+template <typename T>
+inline void ConsoleLogger::unused(T&&) const noexcept
+{
+}
+
+template <typename T>
+inline void ConsoleLogger::logArithmetik(const T value, const char* format) noexcept
```

Comment:
```suggestion
inline void ConsoleLogger::logArithmetic(const T value, const char* format) noexcept
```

Reply:
@MatthiasKillat you don't like German ;)

Reply:
Doch! But there is a time and a place for everything, and this is not the place for German...